projects/ff4j-features/src/app/features-routing.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { FeaturesComponent } from './features.component';
+import { RouterModule, Routes } from '@angular/router';
+
+
+const routes: Routes = [{ path: '', component: FeaturesComponent }];
```

Comment:
path should have features

Reply:
I don't think so Paul, if **features** is there in the path, then from the main app, we have to reference it like `'4200/features/features'`

Reply:
I get your point 👍